src/ArcxAnalyticsSdk.ts

```diff
@@ -109,7 +109,9 @@ export class ArcxAnalyticsSdk {
     }
 
     if (this.sdkConfig.trackUTM) {
-      const searchParams = new URLSearchParams(window.location.search)
+      const searchParams = new URLSearchParams(
+        window.location.search || window.location.hash.split('?')[1],
+      )
```

Comment:
I think you mean the other way around:
```suggestion
      const searchParams = new URLSearchParams(
        window.location.hash.split('?')[1] || window.location.search,
      )
```

Because if you go to a site like https://plausible.io/#/hello?utm_source=test and try the above, it won't work

Reply:
It works. I've tried it

Reply:
I tried it too and somehow it didn't work but tried again and it does work.. I'm not sure, might be a mistake on my end. Apologies!